spec/salestation/web/error_mapper_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+describe Salestation::Web::ErrorMapper do
+  context 'with custom errors' do
```

Comment:
Is `context` needed? There is no alternative to `with custom errors`, so it seems superfluous.